src/ray/gcs/gcs_server/gcs_server.cc

```diff
@@ -115,6 +115,8 @@ void GcsServer::DoStart(const GcsInitData &gcs_init_data) {
   // Print debug info periodically.
   PrintDebugInfo();
 
+  CollectStats();
```

Comment:
Unrelated to this PR. Without this gcs server doesn't collect metrics.

Reply:
Does this mean the existing GCS stats were not working? If so, please add a test that should have covered that.